src/pages/blog/drafts.astro

```diff
@@ -0,0 +1,30 @@
+---
+// © 2025 Functional Software, Inc. dba Sentry
+// SPDX-License-Identifier: Apache-2.0
+
+import ArticleList from "../../components/ArticleList.astro";
+import Blob from "../../components/Blob.astro";
+import Layout from "../../layouts/Layout.astro";
+---
```

Comment:
We should prevent this page from being indexed by search engines, etc. Is that accounted for?

Reply:
I had forgotten about this, sorry. Fixed.